docusaurus/docs/learn/core-concepts/security/authentication/50-external-id-claims.md

```diff
@@ -1,4 +1,6 @@
-# External JWT Signers
+# External ID Claims
+
```

Comment:
It seems weird that there is a 3rd Party CA page and not an External JWT SIgner page. This page was the latter, but it was merged with a meta-concern of mapping identities to external IdPs.

It obscures External JWT SIgners and merges two separate implementations into a meta concern that should either be its own page or not exist.

I think there is space in the documentation for clinically describing what things are and how they work (reference documentation) and another place for describing meta-concerns and how to get things done (admin/user documentation).

I feel that we have lost the reference documentation in the security section with these changes and are now attempting to make it admin/user documentation. I would much rather this page be largely reverted and a new page or section be created for the goals this new page has.

Reply:
I agree we should keep nudging docs toward an organization by audience and intent, e.g., learning (tutorials and explanations) vs. using (references and guides). We've already begun to structure the docs in those ways, but there's still more that we can do. The most helpful model we've stumbled upon so far is this one: https://documentation.divio.com/. How does that model land with you? 

Specifically, we could, as you suggest, organize the reference content separately from concept-level, explanatory articles.